frontend/src/components/ButtonBig.js

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components/macro';
+
+export default function ButtonBig() {
+    return <Button> LET'S PLAY </Button>;
+}
+
+const Button = styled.button`
+    border-radius: 50px;
+    padding: 10px 50px;
+    margin: 8px;
+    background: #c8a1a2;
+    color: white;
+    border: none;
```

Comment:
Think about adding a little shadow to make the button 'pop'

Reply:
added ^^